alerts.go

```diff
@@ -43,26 +44,103 @@ type Alert struct {
 	ClosedAt  int64   `json:"closedAt,omitempty"`
 }
 
-// FindAlerts find alerts
-func (c *Client) FindAlerts() ([]*Alert, error) {
+// FindAlerts find open alerts
+func (c *Client) FindAlerts() ([]*Alert, string, error) {
 	req, err := http.NewRequest("GET", c.urlFor("/api/v0/alerts").String(), nil)
 	if err != nil {
-		return nil, err
+		return nil, "", err
 	}
 	resp, err := c.Request(req)
 	defer closeResponse(resp)
 	if err != nil {
-		return nil, err
+		return nil, "", err
 	}
 
 	var data struct {
 		Alerts []*Alert `json:"alerts"`
+		ID     string   `json:"nextId,omitempty"`
 	}
 	err = json.NewDecoder(resp.Body).Decode(&data)
 	if err != nil {
-		return nil, err
+		return nil, "", err
+	}
+	return data.Alerts, data.ID, err
+}
+
+// FindAlertsByNextID find next open alerts by next id
+func (c *Client) FindAlertsByNextID(nextID string) ([]*Alert, string, error) {
+	v := url.Values{}
+	v.Set("nextId", nextID)
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s?%s", c.urlFor("/api/v0/alerts").String(), v.Encode()), nil)
+	if err != nil {
+		return nil, "", err
+	}
+	resp, err := c.Request(req)
+	defer closeResponse(resp)
+	if err != nil {
+		return nil, "", err
+	}
+
+	var data struct {
```

Comment:
It's better to declare (private) interface to represent alerts and use in those methods.
(Or, define `func (c *Client) findAlertsWithParam(v url.Values)` and share among those methods?)